code/ZAS/Fire.dm

```diff
@@ -256,6 +247,21 @@ turf/proc/apply_fire_protection()
 turf/simulated/apply_fire_protection()
 	fire_protection = world.time
 
+/datum/gas_mixture/proc/get_gas_fuel()
+	var/total_fuel = 0
+	for(var/gasid in gases)
+		var/datum/gas/current_gas = get_gas_by_id(gasid)
+		if(current_gas.gas_flags & IS_FUEL)
```

Comment:
Might want to make these flags into helpers procs (eg ```gas.isFuel()```) instead, for future flexibility (ie if you want to determine whether it's a fuel from multiple properties, or there's more than 16 flags and you want to store them in different places, etc)